documentation/book/con-kafka-mirror-maker-authentication.adoc

```diff
@@ -1,98 +1,55 @@
-The `authentication` property specifies the type of the authentication method which should be used and additional configuration details depending on the mechanism.
-The currently supported authentication types are:
+[id='assembly-kafka-mirror-maker-authentication-{context}']
 
-* TLS client authentication
-* SASL-based authentication using the SCRAM-SHA-512 mechanism
-* SASL-based authentication using the PLAIN mechanism
+= Authentication
 
-You can use different authentication mechanisms for the Kafka Mirror Maker producer and consumer.
+Use the `consumer.authentication` and `producer.authentication` properties to configure authentication for the consumer and producer.
 
-== TLS Client Authentication
+By default, Kafka Mirror Maker will try to connect to Kafka brokers without any authentication.
+
+Use `authentication.type` to set the authentication mechanism, and add additional configuration details depending on the mechanism.
+
+You can use different authentication mechanisms for the Kafka Mirror Maker consumer and producer.
+
+.TLS Client Authentication
+
+Use `type.tls` to configure Kafka Mirror Maker to use TLS authentication.
 
-To use TLS client authentication, set the `type` property to the value `tls`.
 TLS client authentication uses a TLS certificate to authenticate.
 The certificate is specified in the `certificateAndKey` property and is always loaded from a Kubernetes secret.
 In the secret, the certificate must be stored in X509 format under two different keys: public and private.
 
 NOTE: TLS client authentication can be used only with TLS connections.
-For more details about TLS configuration in Kafka Mirror Maker see xref:assembly-kafka-mirror-maker-tls-{context}[].
-
-.An example TLS client authentication configuration
-[source,yaml,subs=attributes+]
-----
-apiVersion: {KafkaApiVersion}
-kind: KafkaMirrorMaker
-metadata:
-  name: my-mirror-maker
-spec:
-  # ...
-  consumer:
-    authentication:
-      type: tls
-      certificateAndKey:
-        secretName: my-source-secret
-        certificate: public.crt
-        key: private.key
-  # ...
-  producer:
-    authentication:
-      type: tls
-      certificateAndKey:
-        secretName: my-target-secret
-        certificate: public.crt
-        key: private.key
-  # ...
-----
-
-== SCRAM-SHA-512 authentication
-
-To configure Kafka Mirror Maker to use SCRAM-SHA-512 authentication, set the `type` property to `scram-sha-512`.
-The broker listener to which clients will connect must also be configured to use SCRAM-SHA-512 SASL authentication.
+
+For more information on configuring TLS encryption and authentication, see:
+
+* xref:proc-configuring-kafka-mirror-maker-tls-{context}[Configuring TLS encryption in Kafka Mirror Maker]
+* xref:proc-configuring-kafka-mirror-maker-authentication-tls-{context}[Configuring TLS client authentication in Kafka Mirror Maker]
+
+.SCRAM-SHA-512 authentication
```

Comment:
Could we keep this as a heading? It is quite important because it allows us to link to the specific section.

```suggestion
=== SCRAM-SHA-512 authentication
```